gonggol/urls.py

```diff
@@ -13,8 +13,8 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
-]
\ No newline at end of file
+    path('main', include('products.urls')),
```

Comment:
main 보다는 product 가 더 적절해 보입니다.
아마도 메인페이지에서 보이는 상품 리스트여서 main 으로 하신 것 같네요.
페이지 단위가 아닌 자원단위로 생각해주시면 좋겠습니다!
```suggestion
    path('product', include('products.urls')),
```

Reply:
넵 수정 했습니다